tests/integration/conftest.py

```diff
@@ -1,16 +1,28 @@
-import pytest
 import os
-from dotenv import load_dotenv
+import pytest
+from azure.identity import DefaultAzureCredential
 from cognite.client import ClientConfig, CogniteClient
 from cognite.client.credentials import OAuthClientCredentials
-from azure.identity import DefaultAzureCredential
-from tests.integration.integration_steps.cdf_steps import remove_time_series_data
-from tests.integration.integration_steps.time_series_generation import generate_timeseries_set
+from cognite.extractorutils.base import CancellationToken
+from cognite.extractorutils.metrics import safe_get
+from cdf_fabric_replicator.metrics import Metrics
+from cdf_fabric_replicator.time_series import TimeSeriesReplicator
+from dotenv import load_dotenv
+from tests.integration.integration_steps.cdf_steps import remove_time_series_data, push_time_series_to_cdf, create_subscription_in_cdf
 from tests.integration.integration_steps.fabric_steps import get_ts_delta_table
+from tests.integration.integration_steps.time_series_generation import generate_timeseries_set
 
 load_dotenv()
 
-TEST_TIMESERIES_TABLE_NAME = "timeseries"
+@pytest.fixture(scope="function")
+def test_replicator():
+    stop_event = CancellationToken()
+    replicator = TimeSeriesReplicator(metrics=safe_get(Metrics), stop_event=stop_event)
+    replicator._initial_load_config(override_path=os.environ["TEST_CONFIG_PATH"])
```

Comment:
Should "TEST_CONFIG_PATH" be added to .env.example?

Reply:
Added